plugins/UM3NetworkPrinting/ClusterUM3OutputDevice.py

```diff
@@ -104,11 +105,9 @@ def requestWrite(self, nodes: List[SceneNode], file_name: Optional[str] = None,
             file_formats = CuraApplication.getInstance().getMeshFileHandler().getSupportedFileTypesWrite()
 
         global_stack = CuraApplication.getInstance().getGlobalContainerStack()
-        if not global_stack:
-            return
-
         #Create a list from the supported file formats string.
-        machine_file_formats = global_stack.getMetaDataEntry("file_formats").split(";")
+        if global_stack:
+            machine_file_formats = global_stack.getMetaDataEntry("file_formats").split(";")
```

Comment:
If `global_stack `doesn't exist, then the `machine_file_formats` is never defined to be used in the next line.

Reply:
Fixed in ac0192f